txn_writer.go

```diff
@@ -349,6 +361,13 @@ func (sw *SafeWriter) Write(root string, sm gps.SourceManager) error {
 		os.RemoveAll(vendorbak)
 	}
 
+	if len(sw.Payload.Manifest.Dependencies) == 0 {
```

Comment:
If the goal is to only print out the example help text when the manifest is empty, then you will also need to test the manifest's `Ovr`, `Ignores` and `Required` fields as well. It may be useful to combine those checks into an `IsEmpty` function on the `Manifest` struct.